CHANGELOG.md

```diff
@@ -138,12 +152,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Add CLI functionality to start covalent with/without Dask
```

Comment:
don't change the CHANGELOG outside the UNRELEASED

Reply:
fixed